diff --git a/amd-ucodegen.c b/amd-ucodegen.c
index 42bdf57..12e426a 100644
--- a/amd-ucodegen.c
+++ b/amd-ucodegen.c
@@ -74,7 +74,7 @@ int validate_mheader(struct microcode_header_amd *h, u16 proc_id)
 	if (proc_id && (proc_id & 0xff00) != (h->processor_rev_id & 0xff00)) {
 		fprintf(stderr, "Processor ID mismatch: 0x%x - 0x%x\n", proc_id,
 			h->processor_rev_id);
-		return 0;
+		//return 0;
 	}
 	if (h->processor_rev_id < 0x6000) {
 		fprintf(stderr, "Bad processor ID 0x%xn", h->processor_rev_id);
@@ -141,6 +141,8 @@ int main(int argc, char **argv)
 	for (i = 0; i < num_entries; i++) {
 		struct microcode_header_amd mheader;
 
+		printf("processing: family: %02x, file: %s\n", ((proc_id >> 12) & 0x0f) + 0x0f, argv[optind + i]);
+
 		fpin = fopen(argv[optind + i], "r");
 		if (!fpin) {
 	    		perror(argv[optind + i]);
